settings.py

```diff
@@ -3,7 +3,25 @@
 
 logger = logging.getLogger(__name__)
 
-LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-sequence: %(message)s"
+LOGGING_FORMAT = "%(message)s"
```

Comment:
Can this be changed to:

```python
LOGGING_FORMAT = "{}".format(message)
```

which is the preferred string formatting method for modern Python.